lib/kino/table.ex

```diff
@@ -89,6 +98,7 @@ defmodule Kino.Table do
     payload = %{
       name: ctx.assigns.info.name,
       features: ctx.assigns.info.features,
+      info: ctx.assigns.info,
```

Comment:
We already send other things from info, so we need just the export:

```suggestion
      export: ctx.assigns.info[:export],
```